curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff75fd4b20c31c23f8802e.md

```diff
@@ -0,0 +1,144 @@
+---
+id: 63ff75fd4b20c31c23f8802e
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Now that we have the data we want, what's next is to populate it on the UI. But the endpoint returns an array of 26 authors, you won't want to populate all of them on the UI at the same time for better performance.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here